models/product-type/src/transformers.ts

```diff
@@ -0,0 +1,21 @@
+import * as AttributeDefinitionDraft from '@commercetools-test-data/attribute-definition';
+import { Transformer } from '@commercetools-test-data/core';
+import type { TProductType, TProductTypeGraphql } from './types';
+
+const transformers = {
+  default: Transformer<TProductType, TProductType>('default', {
+    buildFields: ['attributes', 'createdBy', 'lastModifiedBy'],
+  }),
+  rest: Transformer<TProductType, TProductType>('rest', {
+    buildFields: ['attributes', 'createdBy', 'lastModifiedBy'],
+  }),
+  graphql: Transformer<TProductType, TProductTypeGraphql>('graphql', {
+    buildFields: ['createdBy', 'lastModifiedBy'],
+    addFields: () => ({
+      __typename: 'ProductTypeDefinition',
+      attributeDefinitions: [AttributeDefinitionDraft.random().buildGraphql()],
```

Comment:
This one doesn't use the draft and has the result format
![image](https://user-images.githubusercontent.com/8136478/219147044-f06e0e3b-5ff7-402b-83a3-c8ec37f2afe4.png)

My previous concern was solely around the hardcoded limit/offset/total.

Reply:
🔧  w/ [9abcdf8](https://github.com/commercetools/test-data/pull/198/commits/9abcdf8dbdb349c0cd5f8ae166be84775a538e49)